.github/workflows/docker.yml

```diff
@@ -1,42 +1,96 @@
 name: Build Docker images
 on:
   push:
-    tags:
-      # Do not build on alpha tags!
-      - v[0-9]+.[0-9]+.[0-9]+
+    branches: [main, master]
```

Comment:
I'd revert this back to building on tags - tags + building on a schedule to get new Rust releases is enough.

Reply:
The declared schedule wasn't triggering for me. So I tested out this theory that it had something to do with tags being looked at in the workflow. For some reason the schedule started working after I made this change.

The version is now parsed from the Cargo.toml file so the filter that was specified previously is no longer needed IMO. So I'll change to just building on new tags. Ok?

Reply:
I'd still keep the filter - we don't want to build on alpha tags, which might still happen from time to time.

Reply:
What if someone wants access to a feature that's being worked on / is about to be released? Then they could reference `0.1.337-beta.1-rust-1`.

Note that latest tags will not be built for alpha/beta/rc builds. You will have to reference the alpha/beta tag explicitly.

Reply:
Ok, you convinced me, let's leave it 😁

Reply:
Yay! 😅

Reply:
@LukeMathWalker So time to merge?